spec/javascript/utils/SetupBrowserAndPage.js

```diff
@@ -0,0 +1,35 @@
+import puppeteer from 'puppeteer'
+import { HEADLESS as headless } from '../support/puppeteer/consts'
+
+const IgnoreImageAndCSSLoad = async (page) => {
+  await page.setRequestInterception(true)
+
+  page.on('request', (req) => {
+    if (req.resourceType() === 'stylesheet' || req.resourceType() === 'font' || req.resourceType() === 'image') {
```

Comment:
just to clarify from this morning, do you have a sense of the effect of this piece vs the combination of tests and sharing of browsers between individual tests?

Reply:
This piece appears to remove about 1 minute overall since the majority of the tests can have it be removed, the shared browser sessions removes about the same